Vesper-Agents-Suite/Compound-Collateral-Factor-Agent/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "forta-agent-starter",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
```

Comment:
From `forta-agent@0.019` on, `name` should be the `agentId`, then you can change it to `vesper-6`.
Also you can change the description.